package.json

```diff
@@ -28,7 +28,7 @@
     "serve.prod": "gulp serve.prod --color",
     "start": "gulp serve.dev --color",
     "tasks.list": "gulp --tasks-simple --color",
-    "test": "gulp test --color",
+    "test": "gulp test --color && gulp build.prod --color && gulp build.js.e2e --color && gulp e2e --color",
```

Comment:
Also, we can have `npm run tests.all` which runs both `npm test && npm run e2e`. This command can be used in CI>